internal/pkg/fetch/fetch.go

```diff
@@ -94,21 +90,26 @@ func fetchDocument(sbomURL string, opts *options.FetchOptions) (*sbom.Document,
 	return document, nil
 }
 
-func fetchExternalReferences(document *sbom.Document, backend *db.Backend, opts *options.FetchOptions) error {
+func fetchExternalReferences(document *sbom.Document, backend *db.Backend,
+	opts *options.FetchOptions, outputFile *os.File,
+) error {
```

Comment:
I think this should remain how it was, referencing `opts.OutputFile` and without the unneeded `outputFile` param.